index.html

```diff
@@ -24,20 +24,199 @@
+                                <span>Ruby on Rails</span>
+                                <span>CSS</span>
+                                <span>JavaScript</span>
+                                <span>HTML</span>
+                            </div>
+                            <button class="project-btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="works-card">
+                        <div class="works-card-img"></div>
+                        <div class="works-card-content">
+                            <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                            <div class="works-tags">
+                                <span>Ruby on Rails</span>
+                                <span>CSS</span>
+                                <span>JavaScript</span>
+                                <span>HTML</span>
+                            </div>
+                            <button class="project-btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="works-card">
+                        <div class="works-card-img"></div>
+                        <div class="works-card-content">
+                            <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                            <div class="works-tags">
+                                <span>Ruby on Rails</span>
+                                <span>CSS</span>
+                                <span>JavaScript</span>
+                                <span>HTML</span>
+                            </div>
+                            <button class="project-btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </section>
+
+
+    <!-- About  section -->
+    <section id="about-section">
+        <div class="section-container">
+            <div class="about-title">
+                <h2>About Me</h2>
+                <p class="supporting-text">I can help you build a product,<br>
+                    feature or website Look through <br>
+                    some of my work and experience! If<br>
+                    you like what you see and have a <br>
+                    project you need coded,<br>
+                    don’t hesitate to contact me.
+                </p>
+                <button class="project-btn">Get my resume</button>
+                <div class="about-img"></div>
+            </div>
+          
+            <div class="card-container">
+        <div class="about-card">
+            <div class="icon-container">
+                <img src="./assets/icons/languages.svg" alt="Image Placeholder">
+            </div>
+        
+            <div class="about-card-content">
+                <h3 class="about-card-title">Languages</h3>
+                <div class="about-tags">
```

Comment:
- [ ] I noticed that the project requirements call for the use of "ul" and "li" elements to structure categories in "Work" section and categories in "About myself" section, using "ul" and "li" elements is a good practice when creating lists in HTML. It provides a clear and organized structure to the content, making it easier for users to read and navigate.
Kindly ensure that you use "ul" and "li" elements in those sections meet the project requirements.

<img width="680" alt="image" src="https://user-images.githubusercontent.com/88894525/236070762-6e3d34da-424b-4825-9b4b-2663add5c356.png">